pc_ble_driver_py/observers.py

```diff
@@ -327,10 +327,10 @@ class BLEAdapterObserver(object):
     def __init__(self, *args, **kwargs):
         super(BLEAdapterObserver, self).__init__()
 
-    def on_indication(self, ble_adapter, conn_handle, uuid, data):
+    def on_indication(self, ble_adapter, conn_handle, uuid, char_handle, data):
```

Comment:
I suggest introducing an extra event and add the handle argument there, as an alternative to breaking backward compatibility. It's not ideal, but it's not breaking.
```suggestion
    def on_indication(self, ble_adpater, conn_handle, uuid, data):
    pass

    def on_indication_handle(self, ble_adapter, conn_handle, uuid, attr_handle, data):
```